sdk/storage/azure-storage-blob/src/main/java/com/azure/android/storage/blob/models/BlobSetTierHeaders.java

```diff
@@ -0,0 +1,105 @@
+// Copyright (c) Microsoft Corporation. All rights reserved.
+// Licensed under the MIT License.
+
+package com.azure.android.storage.blob.models;
+
+import com.fasterxml.jackson.annotation.JsonProperty;
+import com.fasterxml.jackson.dataformat.xml.annotation.JacksonXmlRootElement;
+
+/**
+ * Defines headers for SetTier operation.
+ */
+@JacksonXmlRootElement(localName = "Blob-SetTier-Headers")
+public class BlobSetTierHeaders {
+    /*
+     * If a client request id header is sent in the request, this header will
```

Comment:
We have not really talked about if we are going to use `id` or `ID` in the Javadoc for our clients, just making a note on this.